.circleci/config.yml

```diff
@@ -107,13 +107,13 @@ jobs:
           at: ~/picassojs
       - run:
           command: |
-            npx aw -c aw.config.js --mocha.bail false --mocha.reporter mocha-junit-reporter --mocha.reporterOptions.mochaFile ./coverage/junit/junit.xml
+            yarn run test:unit --maxWorkers=2 --coverage 
             if [ ! -z "$COVERALLS_REPO_TOKEN" ]; then
               echo "Uploading coverage results to coveralls.io..."
-              cat ./coverage/unit/lcov.info | npx coveralls
+              cat ./coverage/lcov.info | npx coveralls
             fi
       - store_test_results:
-          path: coverage/junit
+          path: coverage/lcov-report
```

Comment:
I would have expected there to be something [here](https://app.circleci.com/pipelines/github/qlik-oss/picasso.js/2634/workflows/ab78804c-c9e4-41a6-b406-d5bacf369ea4/jobs/24839/tests) given this line. But probably did not work before either..

Reply:
Looks like it requires some extra package for it to produce the test report in correct format (https://circleci.com/docs/collect-test-data#jest)

Reply:
Good catch. Will set it up.